core/src/price_estimation.rs

```diff
@@ -203,13 +203,15 @@ mod tests {
     use super::*;
     use crate::token_info::hardcoded::{TokenData, TokenInfoOverride};
     use anyhow::anyhow;
+    use ethcontract::Address;
     use price_source::{MockPriceSource, NoopPriceSource};
 
     #[test]
     fn price_oracle_fetches_token_prices() {
+        let address = Address::from_low_u64_be(0);
         let tokens = Arc::new(TokenData::from(hash_map! {
-            TokenId(1) => TokenInfoOverride::new("WETH", 18, None),
-            TokenId(2) => TokenInfoOverride::new("USDT", 6, None),
+            TokenId(1) => TokenInfoOverride::new(address, "WETH", 18, None),
```

Comment:
Maybe it makes sense to have a `TokenInfoOverride::for_test(symbol, decimals)` method? Just a suggestion, not a request for a change.